charts/cloudzero-prometheus-agent/README.md

```diff
@@ -29,10 +29,10 @@ The chart can be installed directly with Helm or any other common Kubernetes dep
 
 If installing with Helm directly, the following command will install the chart:
 ```console
-helm install [RELEASE_NAME] cloudzero/cloudzero-prometheus-agent \
-    --set api_key=CLOUDZERO_API_KEY \
-    --set cloudzero.cluster_name=CLUSTER_NAME \
-    --set cloudzero.cloud_account_id=CLOUD_ACCOUNT_ID \
+helm install <RELEASE_NAME> cloudzero/cloudzero-prometheus-agent \
```

Comment:
the chart name is just `cloudzero-agent` now. sorry, I think I forgot to update that when I changed this